### DISCUSSION TITLE
Not sure if bug, so asking here first

### DISCUSSION THREAD

**Discussion by rburke-orthofi** [gitextensions/gitextensions#9297] [Q&A]
Maybe I'm just doing something wrong, but I cannot get a merge tool to be set. I have downloaded a couple. But every time I set one (even though it can find it on its own or I set it manually), it will disappear from the settings after I close the window. Git Extensions will not recognize it, and when I go back into the settings, the Merge tool settings are blank. The diff tool sticks around, but the merge is empty.
![image](https://user-images.githubusercontent.com/78169952/122585133-2224b680-d029-11eb-9f6b-b567d4e5247c.png)
 Please advise if I need to report a bug or if I'm doing something wrong.

**Comment by mstv** [comment]
As this is quite confusing (and I use customized command lines), I just fill in *both* the `Path to ...` and the `... command`.
The ComboBoxes might help to get default values.
(I have not read the git manual in order to find out how it is meant to work.)

**Reply by rburke-orthofi** [reply]
As long as the tool is installed to an anticipated path, it will just work. I have tried a few and they have been auto-populated to the correct path by using the dropdown to pick the one I want to use.
Every time I save, and then try to use the merge tool, it says it can't find it. Go back in to the settings, and the Merge info is just gone.

**Comment by gerhardol** [comment]
If this is older than 3.4 upgrade, there are issues in earlier versions with these settings

**Reply by rburke-orthofi** [reply]
I am on 3.4.3.9999
